sdk/go/common/workspace/plugins.go

```diff
@@ -1504,12 +1504,14 @@ const (
 	ResourcePlugin PluginKind = "resource"
 	// ConverterPlugin is a plugin that can be used to convert from other ecosystems to Pulumi.
 	ConverterPlugin PluginKind = "converter"
+	// ToolPlugin is an arbitrary plugin that can be run as a tool.
+	ToolPlugin PluginKind = "tool"
```

Comment:
Since this public, should the comment mention that this is experimental?

Reply:
I don't think that's needed. common/workspace is _mostly_ seen as internal code not public API.